README.md

```diff
@@ -0,0 +1,9 @@
+# Spy Terminal
+
+The Spy Terminal is a command line program for encrypting secret messages to be decrypted using a shared secret password later
+
+## Getting Started
+
+1. Instatiate a virtual environment to isolated dependencies for this program: `virtualenv venv`
```

Comment:
Python offers virtual environments to create an isolated instance of Python and installed dependencies so that you can manage dependencies and their versions specifically for this program and isolate it from other programs you work on which may share dependencies but with different version requirements.